src/main/java/main/internet/shop/dao/GenericDao.java

```diff
@@ -0,0 +1,19 @@
+package main.internet.shop.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface GenericDao<T, K> {
+    T create(T t);
```

Comment:
```suggestion
    T create(T item);
```
same with other methods